tests/data/brownie-test-config.yaml

```diff
@@ -0,0 +1,43 @@
+networks:
```

Comment:
Same with this YAML re: location.  I want to try to move away from a `data/` folder and keep source info within the tests where it's relevant, I think it's more readable.  Could you move this to a string at the top of `tests/project/test_brownie_config.py` and add it via `yaml.dump` instead of `shutil.copy`?  This should make the test cases much easier to read since we only have to look at one file instead of three.

Reply:
Jep, sounds good!
Will change this and move the fixture.